src/tagstudio/qt/widgets/preview/file_attributes.py

```diff
@@ -142,19 +144,34 @@ def update_stats(self, filepath: Path | None = None, ext: str = ".", stats: dict
             self.dimensions_label.setText("")
             self.dimensions_label.setHidden(True)
         else:
+            filepath_option = self.driver.settings.value(
+                SettingItems.SHOW_FILEPATH, defaultValue="show full path", type=str
+            )
+
+            self.library_path = self.library.library_dir
+            display_path = filepath
+            if filepath_option == "show full path":
+                display_path = filepath
+            elif filepath_option == "show relative path":
+                display_path = Path(filepath).relative_to(self.library_path)
+            elif filepath_option == "show only file name":
+                display_path = Path(filepath.name)
```

Comment:
*(In the context of enums)* This is how the enums would be used in a block of logic such as this:
```suggestion
            filepath_option = self.driver.settings.value(
                SettingItems.SHOW_FILEPATH, defaultValue=ShowFilepathOption.DEFAULT.value, type=int
            )

            self.library_path = self.library.library_dir
            display_path = filepath
            if filepath_option == ShowFilepathOption.SHOW_FULL_PATHS:
                display_path = filepath
            elif filepath_option == ShowFilepathOption.SHOW_RELATIVE_PATHS:
                display_path = Path(filepath).relative_to(self.library_path)
            elif filepath_option == ShowFilepathOption.SHOW_FILENAMES_ONLY:
                display_path = Path(filepath.name)
```